examples/embedding/Makefile

```diff
@@ -6,3 +6,8 @@ hello-embed: hello-embed.o -lmicropython
 
 -lmicropython:
 	$(MAKE) -f $(MPTOP)/examples/embedding/Makefile.upylib MPTOP=$(MPTOP)
+
+clean:
+	rm -rf build
+	rm -f libmicropython.o
+	rm -f hello-embed.o
```

Comment:
Note: normally this should be `$(RM)` but I'm not sure if it's ok to include mkenv.mk here?

Reply:
I included it, and as far as I see it does not contains something dangerous... But I am not an expert in makefiles as well as in particular micropython case.